src/config/dynamic-theme-fixes.config

```diff
@@ -7567,6 +7567,7 @@ img[src*="empty_state_trash"]
 img[src$="google_gsuite"]
 div[tabindex="-1"][role="toolbar"] > div > div > div > svg > path[d^="M10 18h4v-2h-4v2zM3"]
 button[class="ao-search-infolder ao-hint"]
+div > div > div > div > div > div > div:nth-child(2) > div > div:nth-child(1) > div > div > div > div > div:nth-child(1) > span > div > div > div > button > div > svg > path
```

Comment:
This doesn't work for me at all.

I tried to make better selectors for this checks:
![obraz](https://github.com/darkreader/darkreader/assets/56877029/52940036-e26a-465e-9857-2def007ab13a)

and I ended with:
```
button[aria-checked="true"] > div > svg > path
button[aria-checked="true"] > div > span > svg > path
```

Check if my suggestion will work for you too. If make changes in your PR. 😉

Reply:
Yup, this works.